galaxy_ng/app/api/ui/viewsets/my_synclist.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from galaxy_ng.app import models
+from galaxy_ng.app.api import permissions
+from .synclist import SyncListViewSet
+
+log = logging.getLogger(__name__)
+
+
+class MySyncListViewSet(SyncListViewSet):
+    def get_queryset(self):
+        """
+        Returns all synclists for the user.
+        """
+        log.debug('self.request.user: %s', self.request.user)
+        if permissions.IsPartnerEngineer().has_permission(self.request, self):
```

Comment:
If an admin needs to see everyone's synclists, then they can just use the the synclists API, I don't think we need to return all synclists for the admin.